### PR TITLE
[11.0][MIG] stock_batch_picking: Migration to 11.0 (WIP)

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Comment by liebana** [comment]
What's the status of the WIP? How can we help @carlosdauden ?

File: stock_batch_picking/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
```
Comment (nikul-serpentcs) [inline stock_batch_picking/__init__.py]: @carlosdauden Remove Copyright in all `__ini__py` file

File: stock_batch_picking/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'data': [
+        'data/stock_batch_picking_sequence.xml',
+        'data/batch_picking_actions_server.xml',
+        'views/stock_batch_picking.xml',
+        'views/product_product.xml',
+        'views/report_batch_picking.xml',
+        'views/stock_picking.xml',
+        'views/stock_warehouse.xml',
+        'wizard/batch_picking_creator_view.xml',
+        'security/ir.model.access.csv',
```
Comment (nikul-serpentcs) [inline stock_batch_picking/__manifest__.py:31]: @carlosdauden `security` file always first.

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

**Comment by pedrobaeza** [comment]
Please finish this one also fixing Travis tests, @carlosdauden

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@sebalix do you give your approval also?

I will regenerate README by fragments + attend minor comments before merging if so.

**Review by sebalix — APPROVED** [review]
Of course, :+1:

File: stock_batch_picking/i18n/de.po [hunk 3]
```diff
@@ -0,0 +1,396 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * stock_batch_picking
+# 
+# Translators:
+# Rudolf Schnapka <rs@techno-flex.de>, 2017
+# OCA Transbot <transbot@odoo-community.org>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```
Comment (Tonow-c2c) [inline stock_batch_picking/i18n/de.po:10]: Odoo server version should be changed no ?

Comment (pedrobaeza) [reply stock_batch_picking/i18n/de.po:10]: Well, that's informative, and as the same translation file serves for all versions from 9 to 11, it's not needed to change it.

**Pull request merged**